src/main/java/io/pravega/connectors/flink/FlinkPravegaInputFormat.java

```diff
@@ -76,104 +76,112 @@
      * @param controllerURI         The pravega controller endpoint address.
      * @param scope                 The destination stream's scope name.
      * @param streamNames           The list of stream names to read events from.
-     * @param startTime             The start time from when to read events from.
-     *                              Use 0 to read all stream events from the beginning.
      * @param deserializationSchema The implementation to deserialize events from pravega streams.
      */
     public FlinkPravegaInputFormat(
             final URI controllerURI,
             final String scope,
             final Set<String> streamNames,
-            final long startTime,
```

Comment:
This is a breaking change (that I am fine with).

Reply:
Yes. I also think it is ok, since the batch connector is not yet in any official release, correct?